py files doesn't have to be checked is_zipfiles in process_files (#21538)
mistercrunch/Airflow: UPDATING.md

-->

> ### You can't name zip files as ending with .py for DAGs
> 
> It was previously possible to load any files as zip files in the DAGs folder via `zipfile.is_zipfile`.
> 
> Now .py (extension) files are going to be loaded as modules without checking whether it is zipfile. (Less IO)
> If .py file in the DAGs folder were a zip compressed file, processing it would be failed with exception.

Reviewer: ```suggestion
### Zip files in the DAGs folder can no longer have a `.py` extension 

It was previously possible to have any extension for zip files in the DAGs folder. Now `.py` files are going to be loaded as modules without checking whether it is a zip file, as it leads to less IO. If a `.py` file in the DAGs folder is a zip compressed file, parsing it will fail with an exception.
```

Just some minor tweaks.

Author: Thanks fot that. Pushed the commit suggestion.
